src/devices/chihiro/JvsIo.h

```diff
@@ -112,34 +116,53 @@ typedef struct {
 
 typedef struct {
 	jvs_switch_inputs_t switches;
-	jvs_analog_input_t analog[8];
-	jvs_coin_slots_t coins[2];
+	jvs_analog_input_t analog[JVS_MAX_ANALOG];
+	jvs_coin_slots_t coins[JVS_MAX_COINS];
 } jvs_input_states_t;
 
 class JvsIo
 {
 public:
 	JvsIo(uint8_t* sense);
-	void HandlePacket(jvs_packet_header_t* header, uint8_t* payload);
-	size_t SendPacket(jvs_packet_header_t* packet);
+	size_t JvsIo::SendPacket(uint8_t* buffer);
 	size_t ReceivePacket(void* packet);
 	uint8_t GetDeviceId();
 	void Update();
 
 private:
+	const uint8_t SyncByte = 0xE0;

```

Comment:
constants should be defined in upper case, eg: SYNC_BYTE, ESCAPE_BYTE, to make them easily distinguishable from variables.

Reply:
Ah yes. Something for the next work-over

Reply:
Done this too